google/ggrc-core
# GGRC-6077 Display reviewers in export file

# Reviewers are not displayed in exported file

# Steps to test the changes

*Create Control and request review from one or more users.*
*Go to the export and Export all fields for Control.*
*Save exported file and see Reviewers emails in the Reviewers column*

# Solution description

*Expose a simple_property in Reviewable model*
*Create ReviewersColumnHandler column handler to stringify reviewers property*

# Sanity checklist




# PR Review checklist
